functions/Set-SqlTempDBConfiguration.ps1

```diff
@@ -54,7 +54,7 @@ File path to create tempdb data files in. If not specified, current tempdb locat
 .PARAMETER LogPath
 File path to create tempdb log file in. If not specified, current tempdb location will be used.
 
-.PARAMETER Script
+.PARAMETER OutScript
```

Comment:
I'd prefer to see this called [OutputScriptOnly](https://github.com/sqlcollaborative/dbatools/blob/development/functions/Restore-DbaDatabase.ps1#L76), which I think makes it more clear as to what the switch does.